.github/workflows/mutation.yml

```
@@ -26,6 +26,6 @@ jobs:
       os: >-
         ['ubuntu-latest']
       php: >-
-        ['8.1']
+        ['8.0', '8.1', '8.2']
```

```suggestion
        ['8.2']
```

Mutation testing is hard. One version is enough when not version specific code.